doc/api/child_process.md

```diff
@@ -1044,10 +1044,10 @@ added: v0.5.10
 -->
 
 * {boolean} Set to `true` after `subprocess.kill()` is used to successfully
-  terminate the child process.
+  signal the child process.
```

Comment:
Bikeshed : `successfully signal` -> `successfully send a signal to`?

Reply:
Up to me, proposal of @Trott is much better